src/const.h

```diff
@@ -589,4 +589,6 @@ static constexpr int32_t PSTRG_MOUNTS_CURRENTMOUNT = (PSTRG_MOUNTS_RANGE_START +
 
 #define IS_IN_KEYRANGE(key, range) (key >= PSTRG_##range##_START && ((key - PSTRG_##range##_START) <= PSTRG_##range##_SIZE))
 
+#define PREY_SLOTCOUNT 3
```

Comment:
Why?

Reply:
its used a lot of times and I guess its better to have a control of the amount of prey slot players can use, some may change this to use more slots with OTC, Idk.